ambry-server/src/main/java/com.github.ambry.server/AmbryServer.java

```diff
@@ -157,16 +157,22 @@ public void startup() throws InstantiationException {
           networkServer.getRequestResponseChannel(), requests);
       networkServer.start();
 
+      logger.info("Creating StatsManager to publish stats");
```

Comment:
why not avoid creating StatsManager if serverConfig.serverStatsPublishEnabled is not set?